src/Newbe.BookmarkManager/Components/ControlPanel.razor

```diff
@@ -17,43 +17,45 @@
         @if (context.AcceptPrivacyAgreement)
         {
             <FormItem Label="👋 Anonymous Improving">
-                <Switch @bind-Value="@context.ApplicationInsightFeature.Enabled"/>
+                <Switch @bind-Value="@context.ApplicationInsightFeature.Enabled" />
                 <p>👋 Recommended to enable</p>
                 <p>Collecting fully anonymous application performance and error logs helps us to continually strive to provide you with better applications. When enabled, we will collect information about the application's errors, the time spent on each operation, and other data that is not related to personal information. </p>
                 <p>Modifying the options will only take effect after you restart your browser</p>.
             </FormItem>
             <FormItem Label="🌏 Enable Pinyin">
-                <Switch @bind-Value="@context.PinyinFeature.Enabled"/>
+                <Switch @bind-Value="@context.PinyinFeature.Enabled" />
             </FormItem>
             @if (context.PinyinFeature.Enabled)
             {
                 <FormItem Label="Access Token">
-                    <TextArea @bind-Value="@context.PinyinFeature.AccessToken" />
+                    <TextArea @bind-Value="@context.PinyinFeature.AccessToken" OnChange="OnChangeAccessToken" />
                     <p>
-                        <Icon Type="question" Theme="outline"/>
+                        <Icon Type="question" Theme="outline" />
                         You can get an access token from <a href="https://user.newbe.pro/Identity/Account/Manage/AccessToken" target="_blank">user.newbe.pro</a>.
                     </p>
 
+                    <FormItem Label="Expiration Date">
+                        <TimePicker TValue="DateTime?" Disabled="@true" Format="yyyy-MM-dd HH:mm:ss" @bind-Value="@context.PinyinFeature.Expire" />
```

Comment:
Please remove format since this is a global tool. It should be depend on localization